src/BizHawk.Client.EmuHawk/tools/HexEditor/HexColor.Designer.cs

```diff
@@ -41,7 +41,9 @@ private void InitializeComponent()
             this.HexMenubar = new System.Windows.Forms.Panel();
             this.label2 = new BizHawk.WinForms.Controls.LocLabelEx();
             this.HexBackgrnd = new System.Windows.Forms.Panel();
-            this.colorDialog1 = new System.Windows.Forms.ColorDialog();
+			this.label7 = new BizHawk.WinForms.Controls.LocLabelEx();
+			this.Hex00 = new System.Windows.Forms.Panel();
+			this.colorDialog1 = new System.Windows.Forms.ColorDialog();

```

Comment:
Whitespace noise can be cleaned up. If you want I can do it immediately before merge.